app/assets/stylesheets/scaffolds.css.scss

```diff
@@ -0,0 +1,69 @@
+body {
+  background-color: #fff;
```

Comment:
__[scss_lint]__
* [ColorVariable] Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.

<!-- Comment by [SideCI](https://sideci.com) -->